source/2019-09-20-ember-3-13-released.md

```diff
@@ -0,0 +1,150 @@
+---
+title: Ember 3.13 Released
+author: Kenneth Larsen
+tags: Releases, 2019, 3, 3.13
```

Comment:
Needs `Version 3.x` in this list so it shows up under the 3.x link on the blog!

Reply:
omg thanks!

Reply:
Added this as a suggestion so you can trivially auto-apply it!